test/recipes/80-test_ct.t

```diff
@@ -11,7 +11,7 @@ use OpenSSL::Test qw/:DEFAULT srctop_file srctop_dir/;
 use OpenSSL::Test::Simple;
 
 setup("test_ct");
-$ENV{CTLOG_FILE} = srctop_file("test", "ct", "log_list.cnf");
+
 $ENV{CT_DIR} = srctop_dir("test", "ct");
```

Comment:
Why aren't these two env vars centralized?

Reply:
These 2 are only ever set by and read by this test - so there is no possibility of accidentally getting a wrong value from the user's environment. Also since they are only relevant to this test it seems pointless to centralise them.